qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -377,7 +377,7 @@ def launch_custom_cpp_gui(self, interface_name):
         object_name = 'custom-cpp-interface-' + interface_name
         window = find_window(object_name, UserSubWindow)
         if window is None:
-            interface = self.interface_manager.createSubWindow(interface_name)
+            interface = self.interface_manager.createSubWindow(interface_name,self)
```

Comment:
Adding `self` here sets the interface as a child of the main window so it means the interface will always remain on top of workbench - can I just check this is a desired change? I think the fix will work either way.

Reply:
I added this parent link in because of the way the Qt WindowsModal setting works. WindowsModal extends the modal behaviour up the chain of parents link and it seemed sensible to include the main window in the "modal" behaviour for the Indirect Settings window ie both the Interface window and the main workbench window are disabled as well until you close the settings window.

One consequence of  having a parent relationship seems to be that the child stays on top. I'd not specifically decided this was desirable - although having taken a look just now this is how MantidPlot behaves (and the reason is that MantidPlot sets up the equivalent parent link to the main app window in ApplicationWindow.cpp on ~L15735). Please can you let me know if I should leave this alone or change it?

One other thing I've noticed is that the change in this (related) pull request doesn't seem to work:
https://github.com/mantidproject/mantid/pull/27160

At first I thought I'd broken this but the change also doesn't work in last night's nightly build. I can create multiple Indirect, Bayes interfaces in workbench